### DISCUSSION TITLE
Call to undefined method Illuminate\Database\MySqlConnection::collection()

### DISCUSSION THREAD

**Discussion by twigs67** [jenssegers/laravel-mongodb#2134] [Q&A] [answered]
I thought this repo automatically connected with Laravel DB query builder? Is there something else I need to do?

**Comment by divine** [comment, marked as answer]
Hello,

Please give us more information, what are you trying to do and how we can reproduce?

Is this similar to https://github.com/jenssegers/laravel-mongodb/issues/1103?

Thanks!
